### PR TITLE
Modify a change stage trigger feature

### PR DESCRIPTION
**Pull request by volha-pivavarchyk**
| Q                                      | A
| -------------------------------------- | ---
| Branch?                                | "features" for features or enhancements / 3.2 for bug fixes 
| Bug fix?                               | yes/no
| New feature?                           | yes/no
| Deprecations?                          | yes/no
| BC breaks?                             | yes/no
| Automated tests included?              | yes/no
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 

#### Steps to test this PR:
1. Load up [this PR](https://mautibox.com)
2.

### CODE REVIEW

File: app/bundles/LeadBundle/EventListener/PointSubscriber.php [hunk 1]
```diff
@@ -66,13 +54,21 @@ public function onTriggerBuild(TriggerBuilderEvent $event)
             ]
         );
 
+        $stages =$this->stageModel->getUserStages();
+
+        $choices['remove existing stage'] = 0;
```
Comment (adiux) [inline app/bundles/LeadBundle/EventListener/PointSubscriber.php]: is this displayed to the user? then it needs to be translatable. sorry :) 

EN: Remove existing stage
DE: Aktuelle Stage entfernen

Comment (adiux) [reply app/bundles/LeadBundle/EventListener/PointSubscriber.php]: maybe thats a personal choice, but I would put the string in the value
0 => Remove existing stage
...

Or I guess the best one would be to create an object:
[{
label: Remove existing stage
id: 0
}]

At least that is how I would do it in TypeScript.

File: app/bundles/LeadBundle/EventListener/PointSubscriber.php [hunk 2]
```diff
@@ -99,20 +95,32 @@ private function handelChangeTags(TriggerExecutedEvent $event): void
...
+
+        if (0 === $stageId) {
+            $this->leadModel->removeFromStage(
+                $event->getLead(),
+                null,
+                $this->translator->trans('mautic.stage.event.removed.batch')
+            );
 
-        if (null === $stage) {
-            throw new \InvalidArgumentException("Stage for ID $stageId not found");
```
Comment (adiux) [inline app/bundles/LeadBundle/EventListener/PointSubscriber.php:112]: Can we add a logger here? Nice to have

File: app/bundles/LeadBundle/EventListener/PointSubscriber.php (hunk 1 above)
Comment (volha-pivavarchyk) [reply app/bundles/LeadBundle/EventListener/PointSubscriber.php]: This array is added as choises to a symfony form. It should have the structure key => value

**Comment by volha-pivavarchyk** [comment]
I merged 5.x to this branch

**Comment by volha-pivavarchyk** [comment]
The `add-a-change-stage-trigger`branch was directly modified

**Pull request closed without merging**